src/components/Toast/index.js

```diff
@@ -35,14 +36,17 @@ export default class Button extends PureComponent {
     kind: 'default',
   }
 
-  toastRef = React.createRef()
+  containerRef = React.createRef()
```

Comment:
Using an inner container instead of the parent.  This means that the height of the parent can be zero, but we can still calculate the height of the content (absolutely positioned)